vowpalwabbit/cbify.cc

```diff
@@ -139,11 +118,12 @@ void predict_or_learn(cbify& data, base_learner& base, example& ec)
   base.predict(ec);
   //data.probs = ec.pred.scalars;
 
-  uint32_t action = data.mwt_explorer->Choose_Action(*data.generic_explorer, StringUtils::to_string(data.example_counter++), ec);
+  vector<float> pdf = a_s_get_scores(ec.pred.a_s);
```

Comment:
It looks like we still have critical path memory allocation going on.  Remove?

Reply:
done

Reply:
done